ui/dive_tree_view.h

```diff
@@ -62,6 +68,8 @@ public slots:
 private:
     const Dive::CommandHierarchy &m_command_hierarchy;
     FilterMode                    m_filter_mode = kNone;
+    mutable std::vector<uint64_t> m_pm4_draw_call_indices;
```

Comment:
sorry it seems that i am still seeing `mutable` for  m_pm4_draw_call_indices in the latest commit

Reply:
Agh, removed it from filterAcceptsRow but forgot the header. Fixed